tests/functional/pv/pvc_snapshot/test_rbd_cbt_metadata.py

```
@@ -0,0 +1,423 @@
+            previous_snapshot=previous_snapshot,
+        )
+        log.info("Verifier logs:\n%s", logs)
+        assert exit_code == 0, (
+            f"Verifier exited with code {exit_code}. " f"Logs:\n{logs}"
+        )
+        return restored_pvc
+
+    # -- Test 1 ----------------------------------------------------
+
+    @pytest.mark.parametrize(
+        argnames=["volume_mode"],
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `str`

(PT006)